### PR TITLE
feat: wrapped score

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)
image

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "chartjs-plugin-zoom": "2.2.0",
     "consola": "3.2.3",
     "dayjs": "1.11.13",
+    "dom-to-image": "2.6.0",
```
Comment (lukicenturi) [inline frontend/app/package.json:59]: I can't find easy way to do this without 3rd party library.
do you think it's fine? @kelsos 
we can remove it after.

**Review by cleanerzkp — APPROVED** [review]
LGTM

File: frontend/app/src/composables/api/statistics/wrap.ts [hunk 2]
```diff
@@ -4,10 +4,19 @@ import { snakeCaseTransformer } from '@/services/axios-tranformers';
 import { nonEmptyProperties } from '@/utils/data';
 import type { ActionResult } from '@rotki/common';
 
+export const scoresData = [
+  { description: 'Too scared to spend gas (ngmi) 🐣', imageUrl: './assets/images/wrap/wee_wren.png', min: 0, name: 'Wee Wren' },
+  { description: 'Repeating successful trades but playing it safer 🦜', imageUrl: './assets/images/wrap/parrot.png', min: 1000, name: 'Parrot' },
+  { description: 'Standing out in the crypto pool with bold moves 💅', imageUrl: './assets/images/wrap/flamingo.png', min: 4000, name: 'Flamingo' },
+  { description: 'Fearless trader seeking out the juiciest opportunities 💪', imageUrl: './assets/images/wrap/eagle.png', min: 8000, name: 'Defi Eagle' },
+  { description: 'Ultimate degen bird, burning through gas like there\'s no tomorrow 🔥', imageUrl: './assets/images/wrap/pelican.png', min: 10000, name: 'Degen Pelican' },
+];
```
Comment (kelsos) [inline frontend/app/src/composables/api/statistics/wrap.ts:13]: small nitpick here can you just type const?

File: frontend/app/src/components/wrapped/WrappedScore.vue [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+import { useSessionSettingsStore } from '@/store/settings/session';
+import { downloadFileByUrl } from '@/utils/download';
+import { logger } from '@/utils/logging';
+
+const props = defineProps<{
+  score: number;
+  isCurrentYear: boolean;
+}>();
+
+const wrapper = ref();
```
Comment (kelsos) [inline frontend/app/src/components/wrapped/WrappedScore.vue:16]: can we use useTemplateRef('wrapper') here?

File: frontend/app/src/components/wrapped/WrappedConfetti.vue [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+  velocityY: number;
+  x: number;
+  y: number;
+}
+
+// Composable: Use this to manage confetti animation
+function useConfetti() {
+  // Refs
+  const confettiCanvas = ref<HTMLCanvasElement | null>(null);
+  const ctx = ref<CanvasRenderingContext2D | null>(null);
```
Comment (kelsos) [inline frontend/app/src/components/wrapped/WrappedConfetti.vue:17]: is there any reason we can't do this here?
```suggestion
  const confettiCanvas = ref<HTMLCanvasElement>();
  const ctx = ref<CanvasRenderingContext2D>();
```

**Review by kelsos — APPROVED** [review]
ltgm @lukicenturi as always thank you very much :)

**Review by yabirgb — APPROVED** [review]
thanks luki san

**Pull request merged**